src/libcollections/vec.rs

```diff
@@ -1106,12 +1106,36 @@ impl<T: Clone> Vec<T> {
         let len = self.len();
 
         if new_len > len {
-            self.extend(repeat(value).take(new_len - len));
+            self.extend_elem(new_len - len, value);
```

Comment:
Is extend repeat so entrenched that you have to comment on not using it? I just don't really know what to comment. The for loop over a clone that from_elem does is already the simplest way we have.